librz/analysis/esil_interrupt.c

```diff
@@ -78,6 +78,7 @@ RZ_API bool rz_analysis_esil_load_interrupts(RzAnalysisEsil *esil, RzAnalysisEsi
 		}
 		rz_analysis_esil_set_interrupt(esil, intr);
 		i++;
+		free(intr);
```

Comment:
are you sure about this? Isn't `intr` put in the hashtable `esil->interrupts`

Reply:
oh yes, I forget to remove it. Do you have an idea where I could find the source of this function. There is so much macro in sdb.

Reply:
I assume that if the result is false, the intr could be free.